qds_sdk/account.py

```diff
@@ -3,8 +3,55 @@
 """
 
 from qds_sdk.resource import SingletonResource
-
+from qds_sdk.qubole import Qubole
+from qds_sdk.exception import ParseError
+from qds_sdk.util import GentleOptionParser
+from qds_sdk.util import OptionParsingError
+from qds_sdk.util import OptionParsingExit
 
 class Account(SingletonResource):
+    usage = ("account <create> [options]")
+    optparser = GentleOptionParser(usage=usage)
```

Comment:
We are not using optparser for any new code. Please use argparser. Please see the reports api for examples on how to add a new endpoint.